HACKING.rst

```diff
@@ -223,3 +242,10 @@ variable annotations specified in `PEP-526`_ were introduced in Python
 
 .. _PEP-484: https://www.python.org/dev/peps/pep-0484/
 .. _PEP-526: https://www.python.org/dev/peps/pep-0526/
+
+.. [#fixture-list] This list of fixtures (with markup) can be
+   reproduced by running::
+
+     py.test-3 --fixtures -q | grep "^[^ ]" | grep -v no | sed 's/.*/* ``\0``/'``
```

Comment:
what's up with the two trailing `` here?

Reply:
Oop, I think that's holdover from when this was inline in my initial draft. Will fix, thanks!